src/Microsoft.SqlTools.ServiceLayer/QueryExecution/DataStorage/ServiceBufferFileStreamReader.cs

```diff
@@ -216,6 +218,17 @@ private void AssureBufferLength(int newBufferLength)
                 T resultObject = convertFunc(length.ValueLength);
                 result.RawObject = resultObject;
                 result.DisplayValue = toStringFunc == null ? result.RawObject.ToString() : toStringFunc(resultObject);
+                if(setInvariantCultureDisplayValue)
```

Comment:
nit: in this code base the convention is to put a `' '` character between `'f'` and `'('` in `if` statements.

Reply:
done